src/components/GlobalPreferences/Notifications/notification-service-api.js

```diff
@@ -0,0 +1,200 @@
+import {
+  NOTIFICATION_SERVICE_ACCOUNT,
+  NOTIFICATION_SERVICE_LOGIN,
+  NOTIFICATION_SERVICE_VERIFY,
+  NOTIFICATION_SERVICE_SUBSCRIPTIONS,
+  NOTIFICATION_SERVICE_TOKEN_KEY,
+  NOTIFICATION_SERVICE_EMAIL_KEY,
+  EXPIRED_TOKEN_MESSAGE,
+  ExpiredTokenError,
+  UnauthroizedError,
+} from './constants'
+import { getEthNetworkType } from '../../../local-settings'
+
+
+const isTokenExpired = response =>
+  response.statusCode === 401 && response.message === EXPIRED_TOKEN_MESSAGE
+
+const isUnauthorized = rawResponse => rawResponse.status === 401
+
+export const login = async ({ email, dao, network }) => {
+  try {
+    const rawResponse = await fetch(NOTIFICATION_SERVICE_LOGIN, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({ email, dao, network }),
+    })
+    if (!rawResponse.ok) {
+      throw new Error(rawResponse.statusText)
+    }
+  } catch (e) {
+    console.error(e)
+    throw e
+  }
+}
+
+// Verify the short lived email token and fetch a long lived token
+export async function verifyEmailToken(shortLivedToken) {
+  try {
+    const rawResponse = await fetch(NOTIFICATION_SERVICE_VERIFY, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        authorization: shortLivedToken,
+      },
+    })
+
+    if (!rawResponse.ok) {
+      // In case of errors the api will return json payload
+      const response = await rawResponse.json()
+
+      if (isTokenExpired(response))
+        throw new ExpiredTokenError(response.message)
```

Comment:
Let's use braces here and elsewhere in the file :)

Reply:
Makes sense, should we add this to the eslint config https://eslint.org/docs/rules/curly?

Reply:
Yes! I'm surprised prettier doesn't already do this haha.

Reply:
https://github.com/aragon/aragon/pull/924